.rubocop.yml

```diff
@@ -0,0 +1,11 @@
+---
+inherit_from: .rubocop_todo.yml
+
+inherit_gem:
+  voxpupuli-rubocop: rubocop.yml
+
+Style/IfUnlessModifier:
+  Enabled: false
+
+AllCops:
+  TargetRubyVersion: 3.2
```

Comment:
I had hoped we dropped `TargetRubyVersion` from voxpupuli-rubocop by now, but looks like we haven't.